### ISSUE TITLE
RSpotify logo + branding brainstorming

### ISSUE THREAD

**Issue by marioortizmanero** [ramsayleung/rspotify#323]
I've been thinking that it'd be great to have a logo for Rspotify. No rush to get this done, of course, but I wanted to open something similar to get some brainstorming started. It'd be nice if we could get someone with actual artistic skills to do it.

For reference, here's what other projects use:

* [Tekore](https://github.com/felix-hilden/tekore): ![image](https://user-images.githubusercontent.com/25647296/173183291-9b67781b-0658-4079-84bc-0ab2e0639c6c.png). Here's [their discussion about branding](https://github.com/felix-hilden/tekore/issues/121).

* [SpotifyAPI-NET](https://github.com/JohnnyCrazy/SpotifyAPI-NET): ![image](https://user-images.githubusercontent.com/25647296/173183367-ca32c57b-e23e-468f-9570-8249f2df4119.png)

It should be something like Rust + Spotify or Rust + Music. Not a big fan of using Spotify's logo, though, as that will infringe [their guidelines](https://developer.spotify.com/documentation/general/design-and-branding/).

My favorite idea is Ferris with a headset on and a couple subtle music notes around them to make it seem like they are listening to music. I would love that one :)

One last thing: we should probably use consistent casing for the name. It's always been either `rspotify` or `Rspotify`, but in my opinion `RSpotify` would look a bit better. Any thoughts?

**Comment by ramsayleung** [comment]
Great idea! We should give this project a logo, something like Rust + Music would be great.

> One last thing: we should probably use consistent casing for the name. It's always been either rspotify or Rspotify, but in my opinion RSpotify would look a bit better. Any thoughts?

I prefer RSpotify too, because it combines with RS (rust) and Spotify.

**Comment by marioortizmanero** [comment]
> I prefer RSpotify too, because it combines with RS (rust) and Spotify.

We could do that right now, then. I'll upload a PR after #305 is done.

**Comment by NDoolan360** [comment]
By no means do I think I'm "someone with actual artistic skills", however that's not going to stop me from trying.

I wanted to try and get a headset on Ferris, however, no matter how I tried the shape of his head didn't really make that an easy task. After some research apparently crabs don't have ears 🤷‍♂️, so with that justification I gave up on the headset idea.

Here are a couple of simple drafts to throw out there.

Basic Original Ferris:
![rspotify_1](https://github.com/ramsayleung/rspotify/assets/20409659/b67316d4-00df-49f9-957a-287dbd09d5b8)

Basic Cute Ferris:
![rspotify_2](https://github.com/ramsayleung/rspotify/assets/20409659/4fdf9c63-b046-4c36-b68d-acb777dbffa2)

Dark Cute Ferris:
![rspotify_3](https://github.com/ramsayleung/rspotify/assets/20409659/fa6912ac-e196-474b-a2cc-c71dd0f2fa65)


And here are some less basic ones.

Color Dancing Ferris:
![rspotify_4](https://github.com/ramsayleung/rspotify/assets/20409659/1895217e-653e-4eb2-8dc1-1e5cb6375109)

Dancing Ferris with Notes:
![rspotify_5](https://github.com/ramsayleung/rspotify/assets/20409659/6d3ef508-fc1d-4fc3-9543-413222abe413)

Dancing Dark Ferris with Notes:
![rspotify_6](https://github.com/ramsayleung/rspotify/assets/20409659/b371c56d-2f19-4ba6-9254-ac4cdc97c6ed)

**Comment by ramsayleung** [comment]
Cool, great work!

I personally prefer the `Color Dancing Ferris with Notes`. We have three different colors here, we might need to reconcile the colors.

**Comment by NDoolan360** [comment]
Here's a couple quick alts without the solid Spotify green background.
I personally prefer the second dark one. 🌚

Alt 1:
![rspotify_5_alt1](https://github.com/ramsayleung/rspotify/assets/20409659/887e2d93-a1d8-4174-a729-2340a08c0853)

Alt 2:
![rspotify_5_alt2](https://github.com/ramsayleung/rspotify/assets/20409659/09a7a723-7eea-4952-92b6-25c46f272d44)

**Comment by ramsayleung** [comment]
Well, I have to say I am not that into the combination of Ferris and Spotify, as you mentioned that apparently crabs don't have ears. 

Therefore, I design a new logo based on my understanding and imagination, by combining Rust logo(rusty gear) with Spotify, with a rusty library name: **RSpotify**

![Group 1](https://github.com/ramsayleung/rspotify/assets/14180681/349bf7bc-98cb-429d-848c-28520fada5bb)

**Comment by NDoolan360** [comment]
> In my own opinion, the combination is not concise, simple and memorable enough to be a logo.
I agree and I prefer the Rust logo over Ferris in this context and like the way you've combined the two.

Not sure about the font choice for the name though. Maybe we try a font more similar to the one used in the Rust logo.
Something like:
![rspotify_1](https://github.com/ramsayleung/rspotify/assets/20409659/8963c869-cbb5-4bb7-9123-e71311314b21)

Variations:
![rspotify_2](https://github.com/ramsayleung/rspotify/assets/20409659/83e30963-4de5-4bb4-89ae-73b30161c01d)

![rspotify_3](https://github.com/ramsayleung/rspotify/assets/20409659/cdf53df6-5bae-44a4-84fc-be60ef6d9585)

**Comment by ramsayleung** [comment]
I prefer this variation: 

![image](https://github.com/ramsayleung/rspotify/assets/14180681/27775486-0bd9-436d-a7ee-058f123ac9b1)

> Maybe we try a font more similar to the one used in the Rust logo.

Regarding the font family, I stick to the original font style, this font is too formal to be a logo font.
